### ISSUE TITLE
How to remove update listener from airgram

### ISSUE THREAD

**Issue by viet0511** [airgram/airgram#58]
**Issue type:**

[ x] question
[ ] bug report
[ ] feature request
[ ] documentation issue

**Airgram version:**

[x ] `latest`
[ ] `@next`
[ ] `1.1.x` (or put your version here)

How can I remove listener after add update listener ?
Eg: airgram.on(UPDATE.updateAuthorizationState, (ctx, next) => {} >

**Comment by esindger** [comment]
No, you can't remove listeners, because Airgam composes all middleware in a single function. If you need to implement EventEmitter pattern you can easily do it yourself.

```ts
const eventEmitter = new EventEmitter()

airgram.use((ctx, next) => {
  eventEmitter.emit(ctx._, ctx)
  return next()
})

const handler = (ctx) => {
  // some code
}

eventEmitter.on(UPDATE.updateAuthorizationState, handler)
eventEmitter.off(UPDATE.updateAuthorizationState, handler)
```

In this case, you should ensure that the updates are saving in your data store in proper order.

**Comment by Fxlr8** [comment]
How can I create an eventEmitter like this one and also preserve all event and handler types?
